misc/packaging/README.rst

```diff
@@ -3,228 +3,15 @@
-           2. sda
-   1
-   disk type is nvme
-   Sourcing file '/etc/default/grub'
-   Generating grub configuration file ...
-   Found linux image: /boot/vmlinuz-5.3.0-62-generic
-   Found initrd image: /boot/initrd.img-5.3.0-62-generic
-   Found linux image: /boot/vmlinuz-5.3.0-28-generic
-   Found initrd image: /boot/initrd.img-5.3.0-28-generic
-   Added boot menu entry for EFI firmware configuration
-   done
-
-
-
-
-Then install the ACRN-patched kernel package::
-
-   sudo dpkg -i acrn_kernel_deb_package.deb
-
-After that, you're ready to reboot.
-
-.. rst-class:: numbered-step
-
-Boot ACRN Using the Multiboot2 Grub Choice
-******************************************
-
-This time when you boot your target system you'll see some new options:
-
-.. code-block:: console
-   :emphasize-lines: 4
-
-    Ubuntu
-    Advanced options for Ubuntu
-    System setup
-   *ACRN multiboot2
-
-If your target system has a serial port active, you can simply hit
-:kbd:`return` (or wait for the timeout) to boot with this
-``ACRN multiboot2`` choice.
-
-.. important:: If you don't have an active serial port, you'll need to
-   edit the grub configuration a bit by pressing :kbd:`e` (for edit) and
-   remove the ``i915.modeset=0 video=efifb:off`` parameters at the end of
-   the multiboot2 grub command:
-
-   .. code-block:: console
-      :emphasize-lines: 8
-
-      setparams 'ACRN multiboot2 '
-
-          load_video
-          insmod gzio
-          insmod part_gpt
-          insmod ext2
-          search --no-floppy --fs-uuid  --set $uuid
-      multiboot2 /boot/acrn.industry.nuc7i7dnb.bin root=PARTUUID="06d5265d-863a-4bc0-b469-66b5e6b8a9cb" i915.modeset=0 video=efifb:off
-      module2 /boot/vmlinuz-5.4.43.PKT-200203T060100Z Linux_bzImage
-
-   Then press :kbd:`F10` to continue booting
-
-.. rst-class:: numbered-step
-
-Verify ACRN Is Running
-**********************
-
-After the system boots, you can verify ACRN was detected and is running
-by looking at the dmesg log:
-
-.. code-block:: console
-   :emphasize-lines: 3
-
-   acrn@acrn-NUC:~$ dmesg | grep -i acrn
-   [    0.000000] Linux version 5.4.43-PKT-200203T060100Z (root@acrn-NUC7i7DNHE) (gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04)) #1 SMP PREEMPT Thu Jul 16 16:33:58 CST 2020
-   [    0.000000] Hypervisor detected: ACRN
-   [    2.413796] ACRNTrace: Initialized acrn trace module with 4 cpu
-   [    2.413883] ACRN HVLog: Failed to init last hvlog devs, errno -19
-   [    2.413884] ACRN HVLog: Initialized hvlog module with 4 cpu
-
-
-.. comment
-
-   4.python3 compile_iasl.py
-   =========================
-   this script helps compile iasl and cp to /usr/sbin
+Debian packages provide a simple way to build and package ACRN configurations on
+a development computer. You can then copy the packages onto your target system,
+install the packages, and reboot the system with ACRN and an Ubuntu Service VM
+up and running.
+
+By default, Debian packages are created when you build your ACRN hypervisor and
```

Comment:
Let's add more to begin this paragraph:
```
ACRN does not distribute pre-built Debian packages for the hypervisor
or kernel because ACRN and the kernel are configured based on your
specific hardware and scenario configurations, as described in the
:ref:`overview_dev`.  Instead after configuring ACRN to your needs, 
Debian packages are created when you build your ACRN hypervisor and ...
```